Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -297,6 +306,13 @@ AppDelegateViewModelOutputs {
       )
       .skipNil()
 
+    self.findRedirectUrl = openUrl
```

Comment:
ah yes i got confused because `find` sounds like an input and `found` sounds like an output. got em swapped. cool.